docs/api-reference/attribute-manager.md

```diff
@@ -1,6 +1,6 @@
 # AttributeManager Class
 
-The `AttributeManager` class automated attribute allocation and updates.
+The [`AttributeManager`](https://github.com/uber/deck.gl/blob/master/src/lib/attribute-manager.js) class provides automated attribute allocations and updates.
```

Comment:
You are right. We should do 4.0-release. I'll branch 4.0-release after all outstanding PRs got landed today